lib/engine/game/g_system18/map_britain_customization.rb

```diff
@@ -320,8 +320,8 @@ def map_britain_next_round!
             case @round
             when Engine::Round::Stock
               map_britain_stock_round_finished
-              @operating_rounds = @phase.operating_rounds
               reorder_players
+              @operating_rounds = @phase.operating_rounds
```

Comment:
What difference does reordering these lines make?

Reply:
If `reorder_players` happens after `@operating_rounds = @phase.operating_rounds`, the log shows the switch to the OR before the game ends. It doesn't have a functional difference, just esthetic.  It's because I'm sending the `return end_game!` command in the `reorder_players` method.

Reply:
Does this suppress the log message? It comes from `new_operating_round`, which is still being called.
Instead, would sticking an `unless @finished` check on the `new_operating_round` line work?

Reply:
@ollybh I tested this, it functionally makes no difference. In both cases the log output is the same.

Reply:
Could this be a case for some more tests in https://github.com/tobymao/18xx/blob/master/spec/game_state_spec.rb ?

Reply:
Hmmm @ollybh I'm not sure if I was missing it before or what, but yeah now I'm still getting the operating round log message. 

Adding `unless @finished` results in a console error of `Uncaught round_num: undefined method `round_num' for nil` but this works: 

```
        def map_britain_next_round!
          @round =
            case @round
            when Engine::Round::Stock
              map_britain_stock_round_finished
              @operating_rounds = @phase.operating_rounds
              reorder_players
              return if @finished

              new_operating_round
```

What do you think about that?